From a969817f531923ac36d3a3a5915bdf874e95975c Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Wed, 11 Feb 2015 16:17:59 +0800
Subject: [PATCH 0490/5242] MLK-10232-2 gpc: change the number of disply clks

commit  6b4e2a80c2daeac8959351c0b960470c9656bca2 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add vadc clock to dispmix power management.

Signed-off-by: Sandor Yu <R01008@freescale.com>

Moved to drivers/soc/imx/gpc for 4.14

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/soc/imx/gpc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/imx/gpc.c b/drivers/soc/imx/gpc.c
index c37ea9c..672dc6c 100644
--- a/drivers/soc/imx/gpc.c
+++ b/drivers/soc/imx/gpc.c
@@ -468,7 +468,7 @@ static int imx_gpc_old_dt_init(struct device *dev, struct regmap *regmap,
 		disp_clks = 6;
 	} else if (cpu_is_imx6sx()) {
 		pu_clks = 1;
-		disp_clks = 7;
+		disp_clks = 8;
 	} else {
 		pu_clks = GPC_CLK_MAX;
 		disp_clks = 0;
-- 
1.7.9.5

